docs/sources/reference/api/docker_remote_api_v1.16.md

```diff
@@ -1326,6 +1327,11 @@ Create a new image from a container's changes
              "Cmd":[
                      "date"
              ],
+	     "Labels": [
+                     "Vendor=Acme",
+                     "License=GPL",
+                     "Version=1.0"
+             ],
```

Comment:
We need a notion of key and value to support search as I described.  I would rather change the name to Metadata then remove the notion of =